Fix placement of text around button (#1524)
onfido/onfido-sdk-ui: CHANGELOG.md

- Public: Fix `CROSS_DEVICE_START` user analytic event for integrators never being dispatched when user switches to the Cross Device flow
- UI: Update copy in Face Liveness Video intro screen from 25s to 20s to reflect the correct time limit
- Remove old locale key type definitions that are no longer used/exist in code base.

> - UI: Fixed the text placement to be below the button in the `CustomFileInput` component.

Reviewer: `CHANGELOG` note needs to be reworded. Integrators wouldn't necessarily know which screen's UI this is fixing. Also ultimately the fix was applied on `Uploader` component only.
```suggestion
- UI: Fixed the text placement to be below the primary button in the Document Upload screen.
```

Author: Indeed! Applied the changed.

> - UI: Fixed the text placement to be bellow the button in the `CustomFileInput` component.

Reviewer: Typo here
```suggestion
- UI: Fixed the text placement to be below the button in the `CustomFileInput` component.
```

Author: Corrected!

> - UI: Fixed the text placement to be bellow the button, in the `CustomFileInput` component.

Reviewer: Minor: I don't think the comma is needed.
```suggestion
- UI: Fixed the text placement to be bellow the button in the `CustomFileInput` component.
```

Author: Comma removed.
